src/NServiceBus.Transport.RabbitMQ/Routing/DefaultRoutingKeyConvention.cs

```diff
@@ -61,7 +65,7 @@ static bool IsClrType(byte[] a1)
 
         static bool IsNServiceBusMarkerInterface(Type type) => type == typeof(IMessage) || type == typeof(ICommand) || type == typeof(IEvent);
 
-        static readonly byte[] MsPublicKeyToken = typeof(string).Assembly.GetName().GetPublicKeyToken();
+        static readonly byte[] MsPublicKeyToken = typeof(string).Assembly.GetName().GetPublicKeyToken() ?? new byte[0];
```

Comment:
This was to get around a nullable reference types error, maybe there is a better way?